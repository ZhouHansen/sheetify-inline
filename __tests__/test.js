import transform from '../'
import path from 'path'
import fs from 'fs'

describe("sheetify-inline", () => {
  it("test", done => {
    var file = './__tests__/fixtures/basic.css'
    var src = fs.readFileSync(file, 'utf8')

    transform(file, src, {}, function(err, actual){
      if (err) console.log(err)

      expect(actual).toMatchSnapshot()
      done()
    })
  })
})
