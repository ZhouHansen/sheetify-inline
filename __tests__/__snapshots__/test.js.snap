exports[`sheetify-inline test 1`] = `
"div{
  background:url(\'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAASCAYAAABSO15qAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAyRpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMy1jMDExIDY2LjE0NTY2MSwgMjAxMi8wMi8wNi0xNDo1NjoyNyAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvIiB4bWxuczp4bXBNTT0iaHR0cDovL25zLmFkb2JlLmNvbS94YXAvMS4wL21tLyIgeG1sbnM6c3RSZWY9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9zVHlwZS9SZXNvdXJjZVJlZiMiIHhtcDpDcmVhdG9yVG9vbD0iQWRvYmUgUGhvdG9zaG9wIENTNiAoTWFjaW50b3NoKSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDoxQjRFQTg3MDgwQ0MxMUU2OERDRUNDQjcwQzRFOEMyRCIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDoxQjRFQTg3MTgwQ0MxMUU2OERDRUNDQjcwQzRFOEMyRCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOjFCNEVBODZFODBDQzExRTY4RENFQ0NCNzBDNEU4QzJEIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjFCNEVBODZGODBDQzExRTY4RENFQ0NCNzBDNEU4QzJEIi8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+zcpZOgAAAbRJREFUeNp8018onXEcx/GDczEWiow1NVoo8q+ptU5buHAjsyn/QrmZUnSIbSmb48/FRHKxC2wX5oaoCYXcSGTlxg0XjouVCzEuXFhCmvevPqeenp5znnrV85zze77P9/v9fX9hPp/P5XA9lL96folIrOO/dWG4y/lKRZruY5CjIIn2hW6Hl8vwQ18cRr4CPkETWrEYKoM4PEI0evEWy8jGCiaRESpAFP6gDxv6zYtm/MQtMoMFSNHiafTr3gS6xhesIsJauj2Aqe8Cc5jFA/SgRmWYsuIxgAZ7gOeoxDhK8FgBZvAbpWjEPtIxiiRrgA4c4BD1aEeRGvoMI1hTsCv4TSmBWopRjjpUaGDOUI33qFWGZh6y0K3Mjk2ABDVqHrF4jTf4qq/d6LlFJf7DBC4Dg5QLD9rwVPtcqEDvlIFp4J164w28HGiiX503NW/hG15o2pKRp/3/hB0s2Uf5SB02izaV3nfsqXELyrJAGTiehW01yAzPR52HLnzWOTD9mcKuPYB1G83WdOIVTjXGgxjT/0NOx9bpLJhZqNIWejR9H3DiFMDtCn790uCYaTwPtuhegAEA2uhaMYOh++oAAAAASUVORK5CYII=\')
}
"
`;
